src/index.js

```diff
@@ -17,12 +16,14 @@ app.use(express.json())
 app.use(express.urlencoded({ extended: true }))
 app.use(morgan("dev"))
 
+const classesRouter = require("./resources/classes/router");
 const memeberRouter = require("./resources/members/router")
+const trainerRouter = require("./resources/trainers/router")
```

Comment:
Typically you should have your requires/imports at the top of the file. That way you don't have to read every line of your code to see what it imports